MetaMask/metamask-extension
# Fixing truncation and alignment in the network toggle component

## **Description**
Currently there is some truncation issues with long network names in the privacy incoming transactions section. The truncation of the text is not working which results in broken UI for longer network names

This PR adds some styles which resolve the issue as well as adding a story for the network-toggle component for easier revision and review

## **Manual testing steps**

1. Open the storybook build on this PR
2. Search `NetworkToggle` and check the story in small screen sizes
3. Pull this branch
4. Go to Settings > Security & privacy > Incoming transactions section

## **Screenshots/Recordings**

### **Before**


https://github.com/MetaMask/metamask-extension/assets/8112138/9a7028ca-f70a-4833-94db-75e81b791206



### **After**


https://github.com/MetaMask/metamask-extension/assets/8112138/770cb8c7-f8ba-4a40-b941-1115bcbdd83c



## **Related issues**

_Fixes #21369_

## **Pre-merge author checklist**

- [x] I’ve indicated what issue this PR is linked to: Fixes #21369

## **Pre-merge reviewer checklist**
